src/site/tests/support/index.js

```diff
@@ -73,6 +71,10 @@ const updateHTML = files => {
     { from: 'https://www\\.va\\.gov', to: options.hostUrl },
   ];
 
+  return options;
+};
```

Comment:
I had to separate out `options` into its own helper so that I could make certain values available to `siteWideMetadata` a few lines down